### PR TITLE
breakpoints fix and sprites add

### PR DESCRIPTION
**Pull request by felipprodrigues**

### CODE REVIEW

**Review by matheusagcosta — CHANGES_REQUESTED** [review]
@felipprodrigues Remember to add the `.DS_Store` file to the gitignore

File: _src/css/_contact.scss [hunk 1]
```diff
@@ -17,6 +17,10 @@
     color: white;
     font-weight: bolder;
     margin: 0 0 50px;
+
+    @media screen and (min-width: $hd-up) {
+    font-size: 48px;
```
Comment (matheusagcosta) [inline _src/css/_contact.scss]: @felipprodrigues just missing a tab

File: _src/sprites/defs.svg [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
....642-.92-2.357-2.519.536-2.821-.871-3.205-1.607 1.086 1.394 2.718 1.359 3.949.819.683-.3.326-1.064.65-1.343a.75.75 0 00-.407-1.314c-2.314-.255-4.457-1.001-4.457-4.702 0-2.168 1.505-2.362 1.09-3.269-.015-.033-.333-.754-.045-1.849 1.419.262 2.072 1.28 2.753 1.097 1.687-.46 3.544-.46 5.23 0 .704.189 1.207-.801 2.738-1.103.441 1.654-.473 2.058.103 2.677.632.68.953 1.503.953 2.447 0 5.564-4.717 3.957-5.101 5.22a.748.748 0 00.235.792c.61.513.53 1.83.465 2.889-.067 1.098-.125 2.045.482 2.579.214.19.595.393 1.284.253 6.634-2.131 8.83-8.022 8.063-12.917C21.726-2.856 2.296-2.84.184 10.462zm8.27 10.978l.004.505a11.811 11.811 0 01-1.475-.623c.425.109.913.156 1.471.118zm.37-3.7l-.015.08c-.853.252-1.509.001-1.957-.752l-.001-.002c.68.364 1.381.56 1.973.674zM12 2c11.833 0 14.502 16.267 3.469 19.941-.038-.297-.003-.857.021-1.252.058-.951.126-2.059-.213-2.985 5.088-1.059 5.513-6.646 3.554-9.135.243-.952.145-3.189-.729-3.463-.206-.065-1.305-.304-3.437 1.037a11.657 11.657 0 00-5.361 0c-1.064-.667-3.462-1.752-3.922-.6-.534 1.342-.407 2.427-.248 3.03-1.739 2.204-1.218 5.894.534 7.626-.993-.475-2.361-.637-2.656.314-.323 1.037.912.911 1.679 2.804.073.236.208.513.415.788C-1.705 14.54 1.581 2 12 2z'></path>
```
Comment (matheusagcosta) [inline _src/sprites/defs.svg]: @felipprodrigues you don't need the `</path>`, instead you can auto close them like this `<path />`

**Review by matheusagcosta — CHANGES_REQUESTED** [review]

**Comment by felipprodrigues** [comment]
Last changes on portfolio_fix not commited

File: _src/css/_container.scss [hunk 3]
```diff
@@ -17,4 +17,19 @@
     width: 50px;
     border-radius: 50%;
   }
+
+  svg {
```
Comment (matheusagcosta) [inline _src/css/_container.scss:21]: @felipprodrigues let's use a class instead

File: _src/css/_container.scss [hunk 4]
```diff
@@ -17,4 +17,19 @@
...
+    height: 48px;
+
+    * {
+      fill: white;
+    }
+  }
+
 }
+
+svg:not(:root) {
```
Comment (matheusagcosta) [inline _src/css/_container.scss:33]: @felipprodrigues It would be nice to move this rule to a general scss file

**Review by matheusagcosta — CHANGES_REQUESTED** [review]

**Pull request closed without merging**